.buildkite/pipeline.yml

```diff
@@ -151,18 +126,6 @@ steps:
     timeout_in_minutes: 150
 
   # TODO [#124] Uncomment
-  # - label: scheduled delphinet ligo test
-  #   if: build.source == "schedule"
-  #   env:
-  #     TASTY_NETTEST_NODE_ENDPOINT: "http://delphi.testnet.tezos.serokell.team:8732"
-  #   # use another agent for long scheduled jobs
-  #   agents:
-  #     queue: "scheduled"
-  #   commands: *ligo-nettest
-  #   retry:
-  #     automatic:
-  #       limit: 1
-
   # - label: scheduled edonet ligo test
   #   if: build.source == "schedule"
   #   env:
```

Comment:
(one line below, GH doesn't allow me to comment)

If I'm not mistaken I think the port should be changed here as well to `8733`.

Won't make a difference here, but could save some trouble when it will be uncommented.

Reply:
Why? Our edonet node has `8732` as an RPC port

Reply:
Oh right, sorry, got confused a second with local chains